proxies/live/apiproxy/targets/target.xml

```diff
@@ -32,6 +32,18 @@
         <Name>AssignMessage.SetEchoTarget</Name>
         <Condition>(request.header.echo)</Condition>
       </Step> {% endif %}
+     <Step>
+        <Name>Python.DecodeBase64TargetIdentifier</Name>
+        <Condition>request.header.NHSD-Target-Identifier != null</Condition>
+      </Step>
+      <Step>
+        <Name>Javascript.ValidateTarget</Name>
+        <Condition>targetIdentifierDecoded != null</Condition>
+      </Step>
+      <Step>
+        <Name>RaiseFault.InvalidTargetIdentifier</Name>
+        <Condition>request.header.NHSD-Target-Identifier = null or targetIdentifierValid = false</Condition>
```

Comment:
I'm not sure how step conditions are applied but perhaps it would make sense to include something like `!targetIdentifierValid` to make the expression falsy if possible rather than explicitly checking it's false. It should work mostly the same but would also catch the potential scenario where `targetIdentifierValid` is somehow not set by the previous steps.